### DISCUSSION TITLE
Compress / skip time in timing diagram

### DISCUSSION THREAD

**Discussion by SuperiorLQF** [plantuml/plantuml#1690] [Q&A]
In the timing diagram, short events become very small (looking instantaneous) when having long total durations.

As a solution, one could add the possibility to cut out the range between two times, so that timesteps are eluded.

This could work like  the ... "skipping time" ... in sequence diagrams. Start and end of the skipped times would be required. Furthermore no signal change can happen within the skipped time.

**Comment by schiltz3** [comment]
This was an ask back in 2018 as well
https://forum.plantuml.net/7582/compress-skip-time-in-timing-diagram?show=7582#q7582
